src/www/ui/ui-clearing-view.php

```diff
@@ -350,8 +355,18 @@ protected function updateLastItem($userId, $groupId, $lastItem, $currentUploadtr
     $global = GetParm("globalDecision", PARM_STRING) === "on" ? 1 : 0;
     $uploadTreeTableName = $this->uploadDao->getUploadtreeTableName($lastItem);
     $itemBounds = $this->uploadDao->getItemTreeBounds($lastItem, $uploadTreeTableName);
-    $isDecisionWip = $this->clearingDao->isDecisionWip($currentUploadtreeId, $groupId);
-    if ($isDecisionWip || !$global) {
+    if ($global) {
+      $isDecisionWip = $this->clearingDao->isDecisionWip($currentUploadtreeId, $groupId);
+      $hasChangedClearingType = $this->clearingDao->getClearingType($currentUploadtreeId, $groupId, $type);
+      if ($isDecisionWip
+           || !$hasChangedClearingType['scope']
```

Comment:
I just have a question here. Noting that `clearingDao->getClearingType()` can return `false`, won't that make this `|| !$hasChangedClearingType['scope']` check throw a warning (checking array index on bool)??